diff --git a/node_modules/@pgtyped/cli/lib/types.js b/node_modules/@pgtyped/cli/lib/types.js
index 676777d..3f7fd0a 100644
--- a/node_modules/@pgtyped/cli/lib/types.js
+++ b/node_modules/@pgtyped/cli/lib/types.js
@@ -69,6 +69,8 @@ exports.DefaultTypeMapping = Object.freeze({
     jsonb: Json,
     // Bytes
     bytea: Bytes,
+
+    name: String
 });
 function TypeMapping(overrides) {
     return Object.assign(Object.assign({}, exports.DefaultTypeMapping), overrides);
